tests/utils.py

```diff
@@ -32,6 +33,18 @@
 SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
+def is_ip_addr(ipaddr):
+    """Test if given IPA_SERVER_HOST is an IP address."""
+    try:
+        socket.inet_pton(socket.AF_INET, ipaddr)
+    except socket.error:
+        try:
+            socket.inet_pton(socket.AF_INET, ipaddr)
```

Comment:
This is the same call as above. Is that intended?

Reply:
nope, the second one should be AP_INET6.